requirements.txt

```diff
@@ -3,4 +3,5 @@ pandas>=1.0,<1.2.0
 tqdm>=4.36
 pastas>=0.13.0
 contextily
-pyproj
\ No newline at end of file
+pyproj
+adjustText
```

Comment:
if adjust is tested, then fine to leave it here, otherwise it can be left out here.